uiServices/src/testCaseServices.go

```diff
@@ -294,24 +307,27 @@ func deleteTestCase(rw http.ResponseWriter, req *http.Request) {
 }
 func deleteAllTestCases(rw http.ResponseWriter, req *http.Request) {
 	testCasePathDir := getTestCaseHeader(req)
-	logrus.Println("testCasePathDir", testCasePathDir)
+
 	if err := IsHeaderValid(testCasePathDir); err != nil {
+		logrus.Error("No heared path found", err)
 		rw.WriteHeader(http.StatusBadRequest)
 		return
 	}
 
 	files, err := ioutil.ReadDir(testCasePathDir)
 	if err != nil {
-		logrus.Error(err)
+		logrus.Error("Error reading the directory ", err)
+		rw.WriteHeader(http.StatusInternalServerError)
+		return
 	}
 
 	for _, file := range files {
 		filepath := fmt.Sprintf("%s%s", testCasePathDir, file.Name())
```

Comment:
first, we should verify that the files are test cases -- see GET ALL testcases for example

Reply:
```
if testCase.TestName != "" {
				testCases = append(testCases, Case{
					Name:        testCase.TestName,
					Description: testCase.Description,
					HttpMethod:  testCase.HTTPMethod,
				})
			}
```